### ISSUE TITLE
How to pass flag and how to ingore lint

### ISSUE THREAD

**Issue by zfy0701** [sluongng/nogo-analyzer#30]
Hi Guys

Thanks for the great works, I have two questions
1. for some checks, e.g. ST1003, it allow pass custom flags, https://staticcheck.io/docs/configuration/options/#initialisms, I tried:
```
  "ST1003": {
    "analyzer_flags": {
      "initialisms": "GCS"
    }
  }
```
but it reports:
```
compilepkg: nogo: error running analyzers: ST1003: unrecognized flag: initialisms
``
2. there are some situation the lint give false positive and I would need to ignore it, follow this https://staticcheck.io/docs/configuration/#line-based-linter-directives wouldn't work, could please advise?

**Comment by sluongng** [comment]
It's unclear to me how staticcheck set this config. Unless it's passed via Analyzer.Flags, it would not work nicely with nogo. In such case, some additional manual wiring is required. 🤔
